fitbenchmarking/fitbenchmarking
# Make auto display results optional

#### Description of Work

Fixes #474 


#### Testing Instructions

1. Should open browser by default.
2. Should not open browser if FitBenchmarking is run with argument `--dont_open_browser`
3. Should open browser if FitBenchmarking is run with argument `--results_browser`
4. Should not open browser if .ini file contains `results_browser: no`
5. Should  open browser if .ini file contains `results_browser: yes`

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
I will update the options table on the read the docs website once we have confirmed the names of the options
